src/client/oidc-react.ts

```diff
@@ -308,6 +310,45 @@ export function createOidcClient(): Client {
     ...clientFunctions
   };
   bindEvents(manager, { onAuthChange, eventTrigger, setError });
+
+  const pollFunction = async (): Promise<Response | undefined> => {
```

Comment:
The function name should describe better what this function does. Perhaps something like `fetchUserInfo`.

Reply:
True. Changed to userInfoFetchFunction